modules/odc-image-vulnerabilities-breakdown.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+:_content-type: CONCEPT
+[id="odc-image-vulnerabilities-breakdown_{context}"]
+= Image vulnerabilities breakdown
+
+In the developer perspective, the project dashboard shows the *Image Vulnerabilities* link in the *Status* section. Using this link, you can view the *Image Vulnerabilities breakdown* pop-up that includes details regarding vulnerable container images and fixable container images. The severity level of vulnerability is highlighted using the following color icons:
```

Comment:
```suggestion
In the developer perspective, the project dashboard shows the *Image Vulnerabilities* link in the *Status* section. Using this link, you can view the *Image Vulnerabilities breakdown* window, which includes details regarding vulnerable container images and fixable container images. The icon color indicates severity:
```
- IBM SG: pop up adjective Do not use. Use "pop-up", but only if you must specify the type of menu, list, or window.
- Use non-restrictive `, which` instead of `that`.